src/frontend/src/utils/utils.ts

```diff
@@ -398,7 +398,7 @@ export function getCurlCode(
   -d '{"input_value": "message",
   "output_type": "chat",
   "input_type": "chat",
-  "tweaks": ${JSON.stringify(tweaksObject, null, 2)}'
+  "tweaks": ${JSON.stringify(tweaksObject, null, 2)}}'
```

Comment:
This is interesting change without a matching left braces!

Reply:
hello @msmygit, could you clarify your comment for me? I'm not quite sure I understand it. The number of braces seems correct to me, but I might be missing something. Thanks!